dds_cli/account_manager.py

```diff
@@ -194,6 +194,9 @@ def get_user_info(self):
 
             # Get response
             response_json = response.json()
+            for field in response_json["info"]:
+                if isinstance(response_json["info"][field], str):
+                    response_json["info"][field] = rich.markup.escape(response_json["info"][field])
```

Comment:
There's a risk that this will produce a crash. If the `"info"` doesn't exist in the response_json. 

I'd move it out of the try except and add 
```
if info:
    for field in info:
        if instance(response_json["info"][field], str): 
            response_json["info"][field] = rich.markup.escape(response_json["info"][field])
```
under 
```
info = response_json.get("info")
```
on row 216

Reply:
```suggestion
```

Reply:
I can't move it if we want correct values in `LOG.debug`.